massa-execution-exports/src/types.rs

```diff
@@ -4,10 +4,10 @@
 
 use crate::event_store::EventStore;
 use massa_final_state::StateChanges;
+use massa_models::block_id::BlockInfo;
```

Comment:
let's not call it BlockInfo because there are already such names. Maybe `ExecutedBlockInfo` and not put it in Models but in execution_exports

Reply:
Done